release/release_gold.sh

```diff
@@ -328,4 +330,59 @@ function test_boms {
 	fi
 }
 
+function update_release_info_date {
+	if [[ "${_PRODUCT_VERSION}" != *q* ]] ||
+	   [[ "$(echo "${_PRODUCT_VERSION}" | cut -d '.' -f 3)" -eq 0 ]] ||
+	   [[ "$(echo "${_PRODUCT_VERSION}" | cut -d '.' -f 1)" -lt 2024 ]]
+	then
+		lc_log INFO "Skipping the release info update."
+
+		return "${LIFERAY_COMMON_EXIT_CODE_SKIPPED}"
+	fi
+
+	if [[ "$(echo "${_PRODUCT_VERSION}" | cut -d '.' -f 1)" -eq 2024 ]] &&
+	   [[ "$(echo "${_PRODUCT_VERSION}" | cut -d '.' -f 2 | tr -d q)" -le 2 ]] &&
+	   [[ "$(echo "${_PRODUCT_VERSION}" | cut -d '.' -f 3)" -le 11 ]]
+	then
+		lc_log INFO "Skipping the release info update."
+
+		return "${LIFERAY_COMMON_EXIT_CODE_SKIPPED}"
+	fi
```

Comment:
@kiwm, I believe we can remove this code, because if we consider the current scenario the `release_gold.sh` is being executed, e.g. just new releases, then this check may be useless. In light of this, can you please remove it?

Thanks